scripts/ja/OWNERS

```diff
@@ -0,0 +1,13 @@
+# See the OWNERS docs at https://go.k8s.io/owners
+
```

Comment:
Is there a good reason why you need an OWNERS file for the script?

Reply:
Because the script is for `ja` content as the directory name.
I think it is good to get reviews from `ja` reviewers for long-term maintenance.

Reply:
I agree with oomichi's comment, `ja` reviewers can review the content effectively I think.

Reply:
Yes, I understand that you want to manage the changes to the script (which is okay), though I am cautious about adding more OWNERS files to the repo.

Reply:
> Yes, I understand that you want to manage the changes to the script (which is okay)

ah, I see your point.
TBH I am not a member in sig-docs-ja-reviews/owners at all.
I just heard there is some issue of notation fluctuation in `ja` content and created this script to block it as possible to read consistent `ja` document.
I believe sig-docs-ja-reviews/owners will maintain this script to improve the content, and I will do that when necessary.

Anyways thank you for your review @kbhawkey